concent_api/core/payments/sci_backend.py

```diff
@@ -0,0 +1,140 @@
+        payments_list = concent_rpc.get_forced_payments(
+            requestor_address   = requestor_eth_address,
+            provider_address    = provider_eth_address,
+            from_block          = from_block,
+            to_block            = to_block,
+        )
+    elif transaction_type == 'Batch':
+        payments_list = concent_rpc.get_batch_transfers(
+            payer_address   = requestor_eth_address,
+            payee_address   = provider_eth_address,
+            from_block      = from_block,
+            to_block        = to_block,
+        )
+    else:
+        return None
+
+    return payments_list
+
+
+def make_force_payment_to_provider(
+    concent_rpc:            implementation.SCIImplementation    = None,
+    requestor_eth_address:  str                                 = None,
+    provider_eth_address:   str                                 = None,
+    value:                  int                                 = None,
+    payment_ts:             int                                 = None,
+):
+    """
+    Concent makes transaction from requestor's deposit to provider's account on amount 'value'.
+    If there is less then 'value' on requestor's deposit, Concent transfers as much as possible.
+    """
+    assert isinstance(concent_rpc,              implementation.SCIImplementation)   and not None
+    assert isinstance(requestor_eth_address,    str)                                and len(requestor_eth_address) == 42
+    assert isinstance(provider_eth_address,     str)                                and len(provider_eth_address)  == 42
+    assert isinstance(payment_ts,               int)                                and payment_ts   > 0
+    assert isinstance(value,                    int)                                and value        > 0
+
+    requestor_acc_balance = concent_rpc.get_deposit_value(requestor_eth_address)
+    if requestor_acc_balance < value:
+        value = requestor_acc_balance
+
+    tx_hash = concent_rpc.force_payment(
+        requestor_address   = requestor_eth_address,
+        provider_address    = provider_eth_address,
+        value               = value,
+        closure_time        = payment_ts,
+    )
+
+    return tx_hash
+
+
+def is_provider_account_status_positive(
+    concent_rpc:            implementation.SCIImplementation    = None,
+    requestor_eth_address:  str                                 = None,
+    pending_value:          int                                 = None,
+):
+    assert isinstance(concent_rpc,              implementation.SCIImplementation)   and not None
+    assert isinstance(requestor_eth_address,    str)                                and len(requestor_eth_address) == 42
+    assert isinstance(pending_value,            int)                                and pending_value > 0
+
+    requestor_acc_balance = concent_rpc.get_deposit_value(requestor_eth_address)
+
+    return bool(requestor_acc_balance > pending_value)
```

Comment:
This conversion to bool() is not necessary, `>` comparator returns bool by default.

Reply:
Done